man/fido2-crypttab.sh

```diff
@@ -10,3 +10,10 @@ sudo /usr/lib/systemd/systemd-cryptsetup attach mytest /dev/sdXn - fido2-device=
 # If that worked, let's now add the same line persistently to /etc/crypttab,
 # for the future.
 sudo bash -c 'echo "mytest /dev/sdXn - fido2-device=auto" >>/etc/crypttab'
+
+# Depending on your distribution you may need to manually regenerate your initramfs 
+# to be able to use a FIDO2-Device to unlock the partition at boot. (https://unix.stackexchange.com/a/705809)
```

Comment:
We generally try to keep man pages no wider than 80–85 columns.
The example is indented automatically by 11 columns, so examples should be ~70 columns wide.
So please wrap the lines here.

Reply:
I'm not quite sure, I understand completely what you mean by "the example" but I limited all lines to 70 characters / columns. I hope that satisfies this.